app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagConnection.java

```diff
@@ -112,18 +112,18 @@ public static OkHttpClient getClient() {
     }
 
     public static OkHttpClient createClient() {
-        OkHttpClient client = null;
+        OkHttpClient.Builder b = new OkHttpClient.Builder()
+                .readTimeout(45, TimeUnit.SECONDS);
```

Comment:
I liked the idea of setting the timeout for connect, write and read, and even have them as parameters.